google/cloud/storage/internal/curl_streambuf.cc

```diff
@@ -35,7 +35,7 @@ CurlReadStreambuf::CurlReadStreambuf(
 
 bool CurlReadStreambuf::IsOpen() const { return download_.IsOpen(); }
 
-HttpResponse CurlReadStreambuf::Close() { return download_.Close(); }
+HttpResponse CurlReadStreambuf::Close() { return download_.Close().value(); }
```

Comment:
"TODO ... this can throw ...", like below?

Reply:
Done, added bug numbers to the TODOs also.